src/language-server/server.ts

```diff
@@ -61,14 +69,15 @@ workspace.onSchemaTags((params) => {
 });
 
 workspace.onConfigFilesFound(async (params) => {
-  await whenConnectionInitialized;
-
```

Comment:
This was the one place that already had handling for that - we don't need that exception anymore.